G1ANT.Addon.Ocr.Google/G1ANT.Addon.Ocr.Google.Tests/Commands/OcrGoogle.Find/ShouldReturnRectanglePosition_WhenTextIsFound.G1ANT

```diff
@@ -1,13 +1,15 @@
 ﻿addon iexplorer version 2.205.17304.1000
 addon ocrgoogle version 2.205.17304.1000
 
+process.kill iexplore
 ie.open http://digitalnativestudios.com/textmeshpro/docs/rich-text/lowercase-uppercase-smallcaps.png
 window ‴✱Internet Explorer✱‴ style maximize
+delay milliseconds 200
 ocrgoogle.login ♥credential⟦ocrgoogle:jsoncredential⟧
 ocrgoogle.find ‴alice‴ area ♥screen result ♥rectangleResult
 ie.close
 
 test ⊂♥rectangleResult⟦left⟧.GetType() == typeof(Int32)⊃
 test ⊂♥rectangleResult⟦top⟧.GetType() == typeof(Int32)⊃
 test ⊂♥rectangleResult⟦right⟧.GetType() == typeof(Int32)⊃
-test ⊂♥rectangleResult⟦left⟧.GetType() == typeof(Int32)⊃
\ No newline at end of file
+test ⊂♥rectangleResult⟦left⟧.GetType() == typeof(Int32)⊃
```

Comment:
What you test here is that rectangle fields are int. I think what you should check here is that the values are gt 0.

Reply:
No idea not my tests.

Reply:
Check here is that the values are gt 0.